src/python/pants/notes/1.25.x.rst

```diff
@@ -0,0 +1,563 @@
+1.25.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.25.x`` ``stable`` series.
+
+The ``1.25.x`` series brings two major changes to Pants:
+
+1. Deprecating ``globs()``, ``rglobs()``, and ``zglobs()`` in BUILD files.
+2. Adding file args.
+
+Please see https://groups.google.com/forum/#!topic/pants-devel/3nmdSeyvwU0 for more information.
+
+1.25.0rc0 (02/4/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Deprecate `globs`, `rglobs`, and `zglobs` (#9040)
+  `PR #9040 <https://github.com/pantsbuild/pants/pull/9040>`_
+
+* Deprecate `--owner-of` in favor of file arguments (#9050)
+  `PR #9050 <https://github.com/pantsbuild/pants/pull/9050>`_
+
+* Deprecate `--build-file-imports` defaulting to `warn` and having the option `allow` (#9047)
+  `PR #9047 <https://github.com/pantsbuild/pants/pull/9047>`_
+
+* Deprecate `--glob-expansion-failure` in favor of `--files-not-found-behavior` (#9022)
+  `PR #9022 <https://github.com/pantsbuild/pants/pull/9022>`_
+
+* [IntelliJ] Add compiler options and plugins to the output of `export-dep-as-jar` (#9036)
+  `PR #9036 <https://github.com/pantsbuild/pants/pull/9036>`_
```

Comment:
Is this an API change? New feature? Refactoring? cc @wisechengyi

Reply:
new feature. thanks